apps/learner/src/routes/content/[id]/quiz/+page.svelte

```diff
@@ -194,3 +197,26 @@
     </div>
   </div>
 {/if}
+
+{#if showCompletionModal}
+  <div class="z-100 fixed inset-0 flex items-center justify-center bg-slate-950">
+    <Starfield />
+    <div class="flex h-full w-full max-w-5xl flex-col px-4 py-3 transition-all">
+      <div class="flex flex-1 flex-col items-center justify-center">
+        <!-- TODO: placeholder image, to be replaced once confirmed -->
+        <img class="h-128 w-128" src={`${base}/meteor.png`} alt="meteor logo" />
```

Comment:
1. `h-128 w-128` seems quite big, even on wider screen. Maybe can consider `h-96 w-96`?
2. Can explore having responsive height and width as well? For example, `sm:h-64 sm:w-64 h-96 w-96`.
3. Can explore serving different image size using `srcset`? We might need to download multiple images, maybe `256x256` and `384x384`.
4. Another thing to explore is instead of `png`, can we convert it to `webp`? It should save around 25% bandwidth.